server/test/services/export/PdfExporterTest.java

```diff
@@ -150,6 +153,30 @@ public void validatePDFExport_OptionalFileUploadWithoutFile()
     }
   }
 
+  @Test
+  public void validatePDFExport_visibility() throws IOException, DocumentException {
```

Comment:
*suggestion*: it should be clear from the test name what failed, if the test fails. so something like `validatePdfExport_hiddenQuestionAreIncluded` might be clearer

*also nit, optional*: Three letter acronyms should only have the first letter capitalized, but then that wouldn't match the other method names in this file. Up to you which way you want to go.